src/commands/Minion/create.ts

```diff
@@ -10,7 +10,7 @@ import {
 	multiplyBankQuantity
 } from '../../lib/util';
 import createReadableItemListFromBank from '../../lib/util/createReadableItemListFromTuple';
-import Createables from '../../lib/createables';
+import Createables from '../../lib/createables/createables';
```

Comment:
We should probably rename `creatables.ts` to `index.ts` so that this refactoring is transparent outside of the module. Then we can keep this import the same.